### PR TITLE
Browser Refresh button; Update Menu to git pull module changes; Alert…

### PR DESCRIPTION
**Pull request by MonkoftheFunk**
### Added
- Browser Refresh button
- Update Menu to git pull module changes
- Alert Menu to send alerts/notifications to mirror
- Restart will also refresh browser page after 60 seconds
- Successful status updates with 'info' will show alert

*might need to fix some of the tabbing blips that didn't show up

### CODE REVIEW

**Comment by Jopyth** [comment]
[Updating / Refreshing / Send Alerts](https://trello.com/c/lqzh3H52/4-updating-refreshing-send-alerts)

**Comment by MonkoftheFunk** [comment]
I don't know why executing a shell command causes an error command not found, but still executes just fine. I feel like it's a node thing because it doesn't wrap command in quotes. But I'm new.

File: node_helper.js [hunk 1]
```diff
@@ -467,8 +509,56 @@ module.exports = NodeHelper.create({
...
+                        name = this.format(self.configData.moduleData[i].name);
+                        break;
+                    }
+                }
+            }
+        }
+
+        console.log("path:"+path+" name:"+name);
+
+        //todo replace with simple git?
```
Comment (Jopyth) [inline node_helper.js]: Would be great to use simple-git instead, especially since this will become a dependency soon anyway.

**Comment by Jopyth** [comment]
Is the comment about the simple-git command? Or something else?

**Comment by Jopyth** [comment]
I will test and review this on the weekend, thanks! I like the alert send especially.

**Comment by MonkoftheFunk** [comment]
It's about any command, I used pm2 logs to see the errors but the commands still run.

**Comment by MonkoftheFunk** [comment]
Another feature I'm thinking about is an ability to upload and show an image using alert.

**Comment by Jopyth** [comment]
Thanks, merged! I changed some things, mostly visuals/style, so if you want to add new things, such as the image upload, make sure to base your new features on the updated branch.

**Comment by Jopyth** [comment]
As for the command failed, I have the messages now, too. Will make a separate issue.

**Pull request merged**